File: VB/DataPoint.vb
Namespace SideBySideBar2DChart

    Public Class DataPoint

        Public Property XValue As Double

        Public Property YValue As Double

        Public Sub New(ByVal x As Double, ByVal y As Double)
            XValue = x
            YValue = y
        End Sub

        Public Sub New(ByVal value As Double)
            Me.New(value, value)
        End Sub
    End Class
End Namespace

File: CS/DataPoint.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SideBySideBar2DChart
{
    public class DataPoint
    {
        public double XValue { get; set; }
        public double YValue { get; set; }

        public DataPoint(double x, double y) { XValue = x; YValue = y; }
    }
}
